src/OutputWriters/OutputWriters.jl

```diff
@@ -0,0 +1,18 @@
+module OutputWriters
+
+export
+    write_output, read_output,
+    has_reference, has_array_refs,
```

Comment:
Should we export `has_reference` and `has_array_refs`? Doesn't seem like part of API?

Reply:
Probably not. I removed them.